### PR TITLE
Loading animation on "card" element should be stopped on errors

### PR DESCRIPTION
**Pull request by juongithub**
These CSS lines stop the loading animation on "card" element when an error is shown (maybe a code solution would be preferable).

### CODE REVIEW

File: src/scss/main.scss [hunk 1]
```diff
@@ -329,6 +329,17 @@ $curve-easeOutBack: cubic-bezier(0.175, 0.885, 0.32, 1.275);
   }
 }
 
+.braintree-sheet--has-error .braintree-sheet--loading {
+  .braintree-sheet__content {
+    opacity: 1.0!important;
```
Comment (crookedneighbor) [inline src/scss/main.scss]: Do these need important declarations?

**Comment by crookedneighbor** [comment]
Can you provide steps for us to reproduce so we can experience the UI problem?

**Comment by crookedneighbor** [comment]
Can you provide steps for us to reproduce so we can experience the UI problem?

**Comment by juongithub** [comment]
1. generate an auth code with verify=true
2. try to use the card 4000111111111115 
3. by calling requestPaymentMethod() you will see an error but the loading animation is still on

File: src/scss/main.scss (hunk 1 above)
Comment (juongithub) [inline src/scss/main.scss]: actually, "!important" should not be needed, but the dropin script always load the external CSS (#109).

**Comment by juongithub** [comment]
It seems the new commit has solved this problem.

**Comment by crookedneighbor** [comment]
Doesn't seem like this is an issue any longer. Closing.

**Pull request closed without merging**